plans/main.fmf

```diff
@@ -30,6 +40,20 @@ adjust:
             dnf -y copr enable rhcontainerbot/podman-next centos-stream-10
       because: The default epel-10 target doesn't exist yet.
 
+    - when: distro == rhel || distro == centos-stream || distro < fedora-41
+      prepare+:
+        - how: shell
+          order: 40
+          script: dnf config-manager --save --setopt="*:rhcontainerbot:podman-next.priority=5"
+      because: Higher priority for podman-next packages
+
+    - when: distro >= fedora-41
+      prepare+:
+        - how: shell
+          order: 40
+          script: dnf config-manager setopt "*:rhcontainerbot:podman-next.priority=5"
+      because: Higher priority for podman-next packages. Different dnf config-manager usage for Fedora 41+
```

Comment:
Is there no if else syntax? Like this it could be possible to miss a distro accidentally, not that I think it is the case here just looking to make this more robust.

Reply:
let me check if i can simplify this further. Putting this back to draft.

Reply:
@Luap99 simplified a lot by using generic steps to work across all distros. There's still some conditionalizing for enabling centos-stream copr targets on centos-stream test environments and that can't be generalized unless there's changes in how dnf / copr enables targets on rhel/centos-stream envs.